integration-tests/sqls/simple_math_expressions.sql

```diff
@@ -0,0 +1,22 @@
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements.  See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership.  The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License.  You may obtain a copy of the License at
+
+-- http://www.apache.org/licenses/LICENSE-2.0
+
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+SELECT
+  abs(-1.1) as abs,
+  exp(2.0) as exp,
```

Comment:
How are errors handled in the integration test?

I see the following:

```
- 1.1,7.3890560989306502,0.1411200080598672,-0.6536436208636119,-3.380515006246586
?                     --
+ 1.1,7.38905609893065,0.1411200080598672,-0.6536436208636119,-3.380515006246586
+ 
+ 1 rows in set. Query took 0 seconds.
```

So, one small rounding difference and some extra lines. I think we should remove the "x rows in set. Query took n seconds." part for the non-interactive output?

Also, I think we should exit the test with a non-zero error code, so the test will actually fail.

Reply:
turning off timing is done in #324

Reply:
also will try to use pandas to parse csv and use numpy.allclose